.github/workflows/codecov.yml

```diff
@@ -0,0 +1,28 @@
+name: CodeCov
+
+on:
+  push:
+  pull_request:
+    branches: [master]
+
+env:
+  CARGO_TERM_COLOR: always
+  SKIP_WASM_BUILD: 1
+
+jobs:
+  codecov:
+    runs-on: ubuntu-latest
```

Comment:
No, GitHub supports only a very limited set of images.  
The docker does not work because of OS and system library incompatibilities between the docker image and the CI image.